modules/combined/test/tests/contact_verification/hertz_cyl/half_symm_q4/hertz_cyl_half_1deg_template1.i

```diff
@@ -72,6 +73,7 @@
   [./TensorMechanics]
     use_displaced_mesh = true
     save_in = 'saved_x saved_y'
+    extra_vector_tags = 'ref'
```

Comment:
This isn't in the master action yet is it?

Reply:
I think adding
```
  params.addParam<std::vector<TagName>>("extra_vector_tags",
                                        "The extra tags for the vectors this Kernel should fill");
```
to `TensorMechanicsActionBase` input parameters should take care of this. I'm pretty sure tagging, and thus extra_vector_tags, is only applicable to kernels, BCs, and constraints, so the `applyParameters` should take care of the rest.